test/eradius_config_SUITE.erl

```diff
@@ -10,6 +10,7 @@
 -module(eradius_config_SUITE).
 -compile(export_all).
 -include("../include/eradius_lib.hrl").
+-include_lib("eunit/include/eunit.hrl").
```

Comment:
no eunit references in common test please. use the ?match and ?equal macros that we use elsewhere (e.g. https://github.com/travelping/gtplib/blob/master/test/gtp_SUITE.erl#L22)